core/jazz_cognito-admin-authorizer/index.js

```diff
@@ -402,7 +402,7 @@ module.exports.handler = (event, context, cb) => {
        *
        * @method allowAllMethods
        */
-      allowAllMethods: () => {
+      allowAllMethods: function () {
```

Comment:
why?

Reply:
It was throwing TypeError with arrow function. May be it is due to the context attached to "this".